cli/ssh_client.go

```diff
@@ -321,6 +320,14 @@ func (c *sshClient) SendMessages(ctx context.Context, opts options.LoggingPayloa
 	return nil
 }
 
+func (c *sshClient) runManagerCommand(ctx context.Context, managerSubcommand string, subcommandInput interface{}) (json.RawMessage, error) {
```

Comment:
Did you not pull this from master? This was switched in yesterday's PR that was merged.

Reply:
I pulled from master - I had a change of heart after thinking it would be better to separate the `run*Command` functions depending on what struct it's acting on (e.g. `runLoggingCacheCommand` only applies to the LoggingCache implementation, so made it a method on that struct).

Reply:
Oh I didn't se that it is not a method on the `sshClient` struct. Got it!